[2023-12-30T13:02:07.894+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2023-12-30T13:02:08.340+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2023-12-30T13:02:08.348+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-30T13:02:08.751+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): sense_json_file> on 2023-12-19 00:00:00+00:00
[2023-12-30T13:02:08.985+0000] {standard_task_runner.py:57} INFO - Started process 268 to run task
[2023-12-30T13:02:09.078+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'sense_json_file', 'scheduled__2023-12-19T00:00:00+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/dag0001.py', '--cfg-path', '/tmp/tmplm6n593l']
[2023-12-30T13:02:09.287+0000] {standard_task_runner.py:85} INFO - Job 159: Subtask sense_json_file
[2023-12-30T13:02:10.173+0000] {task_command.py:416} INFO - Running <TaskInstance: my_dag.sense_json_file scheduled__2023-12-19T00:00:00+00:00 [running]> on host e1038f319166
[2023-12-30T13:02:12.346+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Javad' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='sense_json_file' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T00:00:00+00:00'
[2023-12-30T13:02:12.627+0000] {logging_mixin.py:154} INFO - Sensing JSON file
[2023-12-30T13:02:12.631+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-30T13:02:12.732+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=sense_json_file, execution_date=20231219T000000, start_date=20231230T130207, end_date=20231230T130212
[2023-12-30T13:02:13.794+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-30T13:02:14.705+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
